pkg/konfluxgen/konfluxgen.go

```diff
@@ -122,17 +162,28 @@ func Generate(cfg Config) error {
 			if ignore {
 				continue
 			}
+			pipeline := DockerBuild
+			for _, r := range fbcImages {
+				if r.MatchString(string(ib.To)) {
+					pipeline = FBCBuild
+					break
+				}
+			}
 
-			applications[appKey][dockerfileComponentKey(c.ReleaseBuildConfiguration, ib)] = DockerfileApplicationConfig{
+			applications[appKey][dockerfileComponentKey(cfg.ComponentNameFunc, c.ReleaseBuildConfiguration, ib)] = DockerfileApplicationConfig{
 				ApplicationName:           cfg.ApplicationName,
```

Comment:
nit: maybe sanitze the appname here too, when we do it for the component name as well :shrug:

Reply:
Done